lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -876,7 +897,7 @@
 
         // See explanation of `getCanonicalLocales`
         const collator_supportedLocalesOf_name = "Intl.Collator.supportedLocalesOf";
-        const collator_supportedLocalesOf_func = tagPublicFunction(collator_supportedLocalesOf_name, function (locales, options) {
+        const collator_supportedLocalesOf_func = tagPublicFunction(collator_supportedLocalesOf_name, function (locales, options = undefined) {
```

Comment:
>locales, options = undefined [](start = 111, length = 28)

if I'm not mistaken, this should change Intl.Collator.supportedLocalesOf.length from 2 to 1 but in master it's already 1 (all engines agree on 1)

Reply:
Same explanation as above